src/Raven.Server/Config/RavenConfiguration.cs

```diff
@@ -103,18 +103,15 @@ private void AddJsonConfigurationVariables(string customConfigPath = null)
         {
             if (string.IsNullOrEmpty(customConfigPath) == false)
             {
-                    _configBuilder.AddJsonFile(customConfigPath, optional: true);
-                // if we were specified a non existing file, don't use the defaults
-                return;
+                if (File.Exists(customConfigPath) == false)
+                    throw new FileNotFoundException("Custom configuration file has not been found.", customConfigPath);
```

Comment:
Fixed the tests to use an empty settings file. We'd like the user to know if he messed up the config file path.